command/validate.go

```diff
@@ -55,7 +55,13 @@ func (c *ValidateCommand) RunContext(ctx context.Context, cla *ValidateArgs) int
 		return 0
```

Comment:
We used to have this line here so that Packer didn't just exit silently.
```suggestion
		c.Ui.Say("Syntax-only check passed. Everything looks okay.")
		return 0
```

Reply:
@nywilken was there a reason you removed that line in f672f5bd9b36012e3ba37377d3ea27f541cd589b ?

Reply:
Good catch 🙂  !

Reply:
hmm... no real reason. I think I went back and forth on the output and ended on only outputting if there is an error.

Reply:
Hm. I guess we can leave it as is for now, but it felt a little weird to have no acknowledgment that it worked.  I guess that is kind of the unix way not to output on success though.